src/index.js

```diff
@@ -779,6 +780,35 @@ function subPublication(context, method, originQualifier) {
     };
 }
 
+/**
+ * Wrap the logger methods (log, info, error, warn) to pass context
+ * @param  {object} context The current object context
+ * @param  {object} logger  The current, unwrapped object logger
+ * @return {object}         The wrapped object logger
+ */
+function contextifyLogger(context, logger) {
+    // Should follow the Logger type defined in ./index.d.ts
+    // Otherwise, the log message will not be contextualized
+    //
+    // for (let method in console) {
+    //   if (typeof console[method] === 'function') {
+    //     ...
+    //
+    // seems overkill
+    ['log', 'info', 'error', 'warn'].forEach((methodName) => {
```

Comment:
Not very elegant :\

Reply:
can you have a look at all possibilities? I think you are only missing `debug` but not sure.